src/js/stores/MarathonStore.js

```diff
@@ -92,6 +96,22 @@ class MarathonStore extends GetSetBaseStore {
         case ActionTypes.REQUEST_MARATHON_GROUP_CREATE_SUCCESS:
           this.emit(MARATHON_GROUP_CREATE_SUCCESS);
           break;
+        case ActionTypes.REQUEST_MARATHON_GROUP_DELETE_ERROR:
+          let groupErrorMessage = action.data;
+          if (!Object.keys(groupErrorMessage).length) {
```

Comment:
there are no views in this PR. i'll handle this issue on https://github.com/dcos/dcos-ui/pull/691